src/main/deploy/swerve/modules/physicalproperties.json

```diff
@@ -0,0 +1,16 @@
+{
+  "optimalVoltage": 12,
+  "wheelGripCoefficientOfFriction": 1.19,
+  "currentLimit": {
```

Comment:
These can probably be bumped up for drive and we could find them doing the wheel slip test, are these current limits for stator current or supply current? More info [here](https://v6.docs.ctr-electronics.com/en/stable/docs/hardware-reference/talonfx/improving-performance-with-current-limits.html)